.travis.yml

```diff
@@ -23,21 +23,31 @@ matrix:
       env:
         - BROWSER=chrome
         - SELENIUM=2.53.6
-    - python: "3.3" # For PR and cron
+        - ROBOTFRAMEWORK=3.0.1
+    - python: "2.7" # For PR and cron
       env:
         - BROWSER=chrome
         - SELENIUM=2.53.6
+        - ROBOTFRAMEWORK=2.9.2
     - python: "2.7" # For PR and cron
       env:
         - BROWSER=chrome
         - SELENIUM=3.0.2
+        - ROBOTFRAMEWORK=2.8.7
+    - python: "3.3" # For PR and cron
+      env:
+        - BROWSER=chrome
+        - SELENIUM=2.53.6
+        - ROBOTFRAMEWORK=3.0.1
     - python: "3.3" # For cron only
       env:
         - BROWSER=firefox
         - SELENIUM=2.53.6
+        - ROBOTFRAMEWORK=3.0.1
 before_script:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - pip install selenium==$SELENIUM
+  - pip install selenium==$ROBOTFRAMEWORK
```

Comment:
Should not play with my hobbies so late in the night and therefore make stupid mistakes like this. Good catch, will fix it.